examples/fscloud/main.tf

```diff
@@ -63,6 +63,7 @@ module "cbr_account_level" {
   target_service_details = {
     "kms" = {
       "enforcement_mode" = "enabled"
+      "target_rg"        = module.resource_group.resource_group_id
```

Comment:
We use a permanent resource group for tests, so this will still be an issue for any test that runs in the resource group. Can we not scope this further to an exact KMS instance ID?

Reply:
cc @jojustin

Reply:
KMS scoped to instance id.